File: Python/SQLAlchemy_ORM/dal/entities/test_datum.py
from sqlalchemy import Column, Integer, String
from dal.data.base import Base

class TestDatum(Base):
    __tablename__ = 'TestData'

    TestID = Column(Integer, primary_key=True)
    TestName = Column(String(50))
    CpuUsage = Column(String(50))
    MemoryUsage = Column(String(50))
    Performance = Column(String(50))
    Language = Column(String(50))

File: C#/EntityFramework_ORM/DAL/Entities/TestDatum.cs
using DAL.Interfaces;

namespace DAL.Entities;

public partial class TestDatum : IEntity
{
    public int Id { get; set; }

    public string Language { get; set; }

    public string TestName { get; set; }

    public string Performance { get; set; }

    public string MemoryUsage { get; set; }

    public string CpuUsage { get; set; }
}
